.github/workflows/docs-latest-public.yaml

```diff
@@ -1,9 +1,7 @@
 name: Docs - Build latest docs and deploy to GitHub pages on main
 
 on:
-  push:
-    branches: ["main"]
-  workflow_dispatch:
```

Comment:
Also why are these triggers gone? I think only the push trigger should be gone. `workflow_dispatch` is useful for those who want to build the docs on branch

Reply:
Good point!